pkg/cli/validate.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	jvspb "github.com/abcxyz/jvs/apis/v0"
+	"github.com/abcxyz/jvs/client-lib/go/client"
+	"github.com/abcxyz/jvs/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// validateCmdOptions holds all the inputs and flags for the validate subcommand.
+type validateCmdOptions struct {
+	config *config.CLIConfig
+
+	token string
```

Comment:
Nit: doc

Reply:
The usage of it is added when it is added as a flag, will the doc here be redundant?

Reply:
The usage is for the CLI user. The docs are for future developers.

Reply:
updated.